zhaquirks/philips/soc001.py

```diff
@@ -0,0 +1,93 @@
+"""Signify SOC001 device."""
+
+from zigpy import types
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import (
+    BinarySensorDeviceClass,
+    ClusterType,
+    EntityType,
+    QuirkBuilder,
+)
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+
+class OnOff(CustomCluster):
+    """Custom On/Off cluster to still allow binding."""
+
+    cluster_id = 6  # 0x0006
+    name = "On/Off cluster"
+    ep_attribute = "on_off_cluster"
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Attribute definitions."""
+
+        on_off = ZCLAttributeDef(
+            id=0x0000,
+            type=types.Bool,
+            is_manufacturer_specific=False,
+        )
```

Comment:
I don't think we should/can do this. We're replacing a ZCL cluster and are also changing the `ep_attribute` here.
IIRC there are places where ZHA expects the `ep_attribute` to be unchanged for a standard ZCL cluster.
Not subclassing `OnOff` (and its `AttributeDefs`) is also not good.